Makefile

```diff
@@ -21,10 +21,22 @@ clean:
 	@rm -rf docs/staging docs/stable
 
 .PHONY: stagingrepo
-stagingrepo: docs/staging/index.yaml $(STAGING_TARGETS)
+stagingrepo: $(STAGING_TARGETS) | docs/staging/index.yaml
 
 .PHONY: stablerepo
-stablerepo: docs/stable/index.yaml $(STABLE_TARGETS)
+stablerepo: $(STABLE_TARGETS) | docs/stable/index.yaml
+
+.PHONY: publish-ci
+publish-ci:
+	@git remote add publish git@github.com:mesosphere/charts >/dev/null 2>&1 || true
+	@git branch -d master >/dev/null 2>&1 || true
+	@git checkout -B master
+	@make all
+	@git add .
+	@git commit -m 'docs: publish repo updates'
+	@git push -f publish $(PULL_REQUEST_ID)
+	@git request-pull master ./
```

Comment:
the final intention would be to open a PR.

Reply:
I don't want a PR, I want a push. The review was already done for the dev branch. I would call master anything else but github pages will only publish from a branch named "master".

Reply:
That's why all PR's are supposed to be against dev.